### DISCUSSION TITLE
Panorama: Maximum width

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#391] [Suggestion Box]
It would be nice to be able to supply Panorama with a maximum width in some way, as it can make posts quite difficult to read with a very wide browser window. I personally would enjoy being able to specify this as the width of the actual post content (for example, 640px would perfectly match a 1280px wide image on a 2x retina display). I did try to implement this, but it seemed difficult to do elegantly and in a way that looks intentional, at least with my knowledge of CSS.

Not sure if this should be another discussion post, but relatedly and/or instead, I feel as though having some padding on the sides of the expanded dashboard once the browser width reaches a certain size makes Panorama look much more like an intended site layout rather than looking somewhat broken. I could imagine some CSS math setting the post container width working nicely here, like the maximum of (some breakpoint in pixels) and (some percentage of the viewport width), but I'm not sure what numbers would feel most natural.

**Comment by arcanetrivia** [comment]
I agree. I like something to use **more** of the width available at 1920 px, because so much of it seems like wasted space especially when I have dash sidebar elements all hidden (don't start me on gripes about individual blog themes, ugh), but using **all** of it tends to be a bit much.
